dashboard/s3.py

```diff
@@ -36,7 +36,7 @@ def __get_images(self):
         )
 
         for image in images['Contents']:
-            print image['Key']
+            print(image['Key'])
```

Comment:
This gets displayed on the terminal while it fetches app images, to give you a sense of progress. So I'll defer to @andrewkrug on whether we still need this.